File: opcua-webservice-spring/core/src/main/java/org/opcfoundation/webserver/types/message/ReadVariableAttributeRequest.java
package org.opcfoundation.webserver.types.message;

import org.opcfoundation.webserver.types.UaObjectId;

public class ReadVariableAttributeRequest {
    private final UaObjectId objectId;
    private final String variableId;

    public ReadVariableAttributeRequest(
            UaObjectId objectId,
            String variableId)
    {
        this.objectId = objectId;
        this.variableId = variableId;
    }

    public UaObjectId getObjectId() {
        return objectId;
    }

    public String getVariableId() {
        return variableId;
    }
}

File: opcua-webservice-node/src/server/service/message/reactiveobject/ReadVariableAttributeRequest.ts
import { UaObjectId } from "../../../types";

export class ReadVariableAttributeRequest {
    private readonly _objectId: UaObjectId;
    private readonly _variableId: string;

    constructor(
        objectId: UaObjectId,
        variableId: string,
    ) {
        this._objectId = objectId;
        this._variableId = variableId;
    }

    get objectId(): UaObjectId {
        return this._objectId;
    }

    get variableId(): string {
        return this._variableId;
    }
}